toonz/sources/toonz/xshcellviewer.cpp

```diff
@@ -1211,51 +1312,41 @@ void CellArea::drawLevelCell(QPainter &p, int row, int col, bool isReference) {
 
   // paint cell
   p.fillRect(rect, QBrush(cellColor));
-  // paint side bar
-  p.fillRect(QRect(x, y, 7, RowHeight), QBrush(sideColor));
 
-  if (yetToCleanupCell)
+  drawDragHandle(p, xy, sideColor);
+
+  if (yetToCleanupCell)  // ORIENTATION: what's this?
```

Comment:
Fixed. Modified the top-left of bounding box that makes this effect to move along x or y axis depending on orientation.